test/functional/wallet_listreceivedby.py

```diff
@@ -167,5 +169,117 @@ def run_test(self):
         balance = self.nodes[1].getreceivedbylabel("mynewlabel")
         assert_equal(balance, Decimal("0.0"))
 
+        self.run_include_coinbase_test()
```

Comment:
I don't see much benefit in splitting this out into a separate function, unless you also restructure the `run_test()` function to be:

```python
    def run_test(self):
        self.run_listbyreceived_test()
        self.run_include_coinbase_test()
```